### PR TITLE
add new file handler for production logger

### PR DESCRIPTION
**Pull request by alexxi0213**
## Summary of changes

- use logging dictConfig to make logging configurable

## How to Test
- create a setting config like `.dev.settings.cfg` and add below config
- set the `'PRODUCTION_LOGGING_CLASS' = 'WatchedFileHandler'` in you config
- see logs in your log file

/fyi: @zhangjustin @mark-juwai  pls review

### CODE REVIEW

**Comment by mark-juwai** [comment]
So if we want to use `WatchedFileHandler` for our rpc servers, we have to put the entire logging config file in rpc config file?

**Comment by alexxi0213** [comment]
@mark-juwai yes, but we can use the template as I added only modify the handler name

**Comment by mark-juwai** [comment]
From my point of view, it's kind of mess the config file up, so many useless (we don't care) messages. I prefer to just add one variable for the logging handle. It's possible, right?

**Comment by alexxi0213** [comment]
@mark-juwai configurable logger requires a configuration imagine if next time we need a configurable format we need to add another variable.

**Comment by mark-juwai** [comment]
As a framework, I think it should be can be used as simple as possible.
We don't have many requirements since `Zask` been finished, before this one. Even there will be a requirement, I don't think it's a problem, that's how framework maintains.

**Comment by alexxi0213** [comment]
The problem is not about `zask` itself, the problem is some module use `zask` rely on zask's configuration management and logger management, like `scripts`, so any requirement from any module use `zask` will lead to adding code or variable in `zask`.

**Comment by mark-juwai** [comment]
@juwai/platform-all Appreciate your thoughts. :)

**Comment by alexxi0213** [comment]
@mark-juwai @zhangjustin updated, pls review

**Comment by mark-juwai** [comment]
https://github.com/juwai/zask/blob/master/zask/__init__.py#L3

Should we increase the version number here?

**Comment by alexxi0213** [comment]
@mark-juwai oh, yes, I did not know how to release, let me check

File: zask/logging.py [hunk 1]
```diff
@@ -35,9 +35,13 @@ def debug_handler():
 
 
 def production_handler(config):
-    handler = RotatingFileHandler(config['ERROR_LOG'],
-                                  maxBytes=1024 * 50,
-                                  backupCount=5)
+    if config.get('PRODUCTION_LOGGING_CLASS') == 'WatchedFileHandler':
```
Comment (mark-juwai) [inline zask/logging.py:38]: ? `PRODUCTION` is a little confusing, maybe just call it `LOGGING_HANDLER`?

Comment (alexxi0213) [reply zask/logging.py:38]: There is another setting: `PRODUCTION_LOGGING_LEVEL`, I try to maintain consistency.

**Review by mark-juwai — APPROVED** [review]

**Comment by alexxi0213** [comment]
@zhangjustin pls help to approve

**Review by zhangjustin — APPROVED** [review]

**Pull request merged**